addons/zeus/functions/fnc_ui_editableObjects.sqf

```diff
@@ -0,0 +1,67 @@
+/*
+ * Author: Fisher, SilentSpike
+ * Updated all/local curator with objects in the module radius.
+ *
+ * Arguments:
+ * 0: editableObjects controls group <CONTROL>
+ *
+ * Return Value:
+ * None
+ *
+ * Public: No
+ */
+
+#include "script_component.hpp"
+
+params ["_control"];
+
+disableSerialization;
```

Comment:
does nothing in unscheduled env. should be removed if these module scripts do indeed run unscheduled. otherwise ignore this